README.md

```diff
@@ -37,8 +37,9 @@ SDK supports them in a way that is compatible with FirebaseUI.
 7. [Developer Setup](#developer-setup)
 8. [Cordova Setup](#cordova-setup)
 9. [React DOM Setup](#react-dom-setup)
-10. [Known issues](#known-issues)
-11. [Release Notes](#release-notes)
+10. [Angular 2 Setup](#angular-2-setup)
```

Comment:
It should just be called "Angular", not "Angular 2", especially since it's on v6 with v7 RCs.